### DISCUSSION TITLE
Great program!

### DISCUSSION THREAD

**Discussion by CamsShaft** [horsicq/XELFViewer#47] [General]
Just had to write to say this is an amazing program you've made! I've barely scratched the surface but I'm already starting to figure out how to read elf files instead of a bunch of nonsense. It definetly helps to better understand more of the processes with my phone. Thank you so much and I'll be recommending it in any forums I visit. Cheers!

**Comment by horsicq** [comment]
Thank you! :)

**Reply by ericshufro** [reply]
I second this.  I use it under windows and its very helpful.  Can you change the string loading limit from 10,000 to 1,000,000?  Computers are fast enough and 10k loads in a couple of seconds.  My binary has a lot more and most gets truncated out of the list.  Thanks again!
